content/en/blog/2023/traffic-for-ambient-and-sidecar/index.md

```diff
@@ -49,7 +49,7 @@ All network traffic coming in/out of the pod with ambient mode enabled will go t
 
 According to above diagram, the details of network traffic path is demonstrated as below:
 
-**(1) (2) (3)**  Request traffic of the `sleep` service is sent out from the `veth` of the `sleep` pod where it will be marked and forwarded to the `istioout` device in the node by following the iptables rules and route rules. The `istioout` device in the node A is a `[geneve](https://www.rfc-editor.org/rfc/rfc8926.html)` tunnel, and the other end of the tunnel is `pistioout`, which is inside the ztunnel pod on the same node.
+**(1) (2) (3)**  Request traffic of the `sleep` service is sent out from the `veth` of the `sleep` pod where it will be marked and forwarded to the `istioout` device in the node by following the iptables rules and route rules. The `istioout` device in the node A is a [Geneve](https://www.rfc-editor.org/rfc/rfc8926.html) tunnel, and the other end of the tunnel is `pistioout`, which is inside the ztunnel pod on the same node.
```

Comment:
Not sure if in or inside works better.
```suggestion
**(1) (2) (3)**  Request traffic of the `sleep` service is sent out from the `veth` of the `sleep` pod where it will be marked and forwarded to the `istioout` device in the node by following the iptables rules and route rules. The `istioout` device in node A is a [Geneve](https://www.rfc-editor.org/rfc/rfc8926.html) tunnel, and the other end of the tunnel is `pistioout`, which is inside the ztunnel pod on the same node.
```

Reply:
For step, #6, I'm wondering if changing `At last` to `Finally` makes sense.